frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -113,40 +113,40 @@ export default function ProofOfCompletedService_Web({ data }) {
           listed in section 3 by mail or commerical courier service to:
         </p>
         <TextInput
-          dataKey="proofofcompletedservice_person.fullname"
+          dataKey="person.proofOfCompletedServiceMailFullname"
           label="Name of Addressee"
         />
         <TextInput
-          dataKey="proofofcompletedservice_person.addressStreet"
+          dataKey="person.proofOfCompletedServiceMailAddressStreet"
           label="Street Address"
         />
         <TextInput
-          dataKey="proofofcompletedservice_person.addressCity"
+          dataKey="person.proofOfCompletedServiceMailAddressCity"
```

Comment:
Same here about this being specific to the case, not the person

Reply:
Corrected to use `case` data key.